### ISSUE TITLE
(03.05. Thr. 15:00) Feature Learning Seminar 3

### ISSUE THREAD

**Issue by cth127** [AIML-K/bamboo-forest#278]
**Abstract**:
From the previous lecture, we observed that removing Gram-invariant components from the original gradient can improve optimization performance. This suggests that the weight Gram matrix lies at the core of effective feature learning.

In this lecture, we define feature learning starting from a primitive observation: a neural network is trained to linearly predict (or classify) nonlinear input data. To formalize this perspective, we introduce a novel concept, **Target Linearity**. We (1) analyze its relationship to the weight Gram matrix, (2) develop a mathematical framework for its analysis, and (3) study how it evolves during gradient descent training.

**Lecturer**: Taehun Cha

**Time/Location**: Mar. 05. (Thr.) 15:00 (before the Townhall meeting) / ASB 526

**Prerequisites** (Optional but would be very helpful):
- "Coefficient of determination" (aka R^2), Wikipedia, [Link](https://en.wikipedia.org/wiki/Coefficient_of_determination)
